sys/test.txt

```diff
@@ -187,3 +187,12 @@ syz_end_var_struct {
 	f1	const[0x42, int32be]
 	f2	flags[syz_end_flags, int64be]
 } [packed]
+
+# Const buffer type
+
+syz_test$bufconst(a0 ptr[in, syz_bufconst])
+
+syz_bufconst {
+	f0	bufconst["wlan0"]
```

Comment:
I agree with you. Could we define like this : name	bufconst["wlan0", buflen[64]]? The remaining bytes will be filled with zero.